src/views/Root.js

```diff
@@ -13,9 +13,10 @@ function Root() {
         <Route path="/" element={<MainLayout />}>
           {/* public routes */}
           <Route index exact path="/" element={<LandingPage />} />
-          <Route path="confirm/:token" element={<EmailConfirmation />} />
           <Route path="login" element={<LoginPage />} />
           <Route path="register" element={<RegisterPage />} />
+          <Route path="validate" element={<Message type="validate" />} />
```

Comment:
wait nie wiedzialem ze jest cos takiego jak element...
to dziala?
https://v5.reactrouter.com/web/api/Route

Reply:
Tak, doszło to w React Router 6